userspace/libsinsp/test/filterchecks/k8s.cpp

```diff
@@ -194,15 +191,11 @@ TEST_F(sinsp_with_test_input, K8S_FILTER_check_fields_value_with_no_labels)
 	container_info->m_type = CT_DOCKER;
 	container_info->m_lookup.set_status(sinsp_container_lookup::state::SUCCESSFUL);
 	container_info->m_labels = container_labels;
+	container_info->m_pod_sandbox_labels = pod_sandbox_labels;
 	container_info->m_container_ip = ip;
-	container_info->m_pod_cniresult = cni_json;
+	container_info->m_pod_sandbox_cniresult = cni_json;
 	m_inspector.m_container_manager.add_container(std::move(container_info), init_thread_info);
 
-	auto sandbox_container_info = std::make_shared<sinsp_container_info>();
```

Comment:
This test now also clearly shows that we have all we need in the container, no need for extra pod sandbox container lookup from the container cache anymore.